src/components/common/Footer.vue

```diff
@@ -139,7 +110,9 @@ export default {
   }
   .footer-logo-text{
     font-size:18px;
-    font-family:SourceSansPro-Regular;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Hiragino Sans GB',
```

Comment:
It would be greate to use css variable for font-family

Reply:
Yes, it really would be really convenient. But i'm not sure, that embedded vue styles support scss